core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.html

```diff
@@ -79,8 +79,23 @@
   side-navigation-bar .oppia-sidebar-menu-icon.fa {
     font-size: 24px;
   }
-
+  #classroom-submenu {
+    display: none;
+    padding: 0 0 1em 18px;
+  }
 </style>
+
+<script>
+  function toggleClassroomSubmenu() {
+    var x = document.getElementById("classroom-submenu");
+    if (x.style.display === "none") {
+      x.style.display = "block";
+    } else {
+      x.style.display = "none";
+    }
+  }
+</script>
```

Comment:
Please avoid having logic inside script tags. Also, why do we need this?

Reply:
This function toggles the submenu i.e. if the submenu is open then on click it will be closed and vice versa. Okay, then I will remove this js code enclosed in script tag to `side-navigation-bar.directive.ts` file. Thanks!

Reply:
Yep but you seem to be adding only a "classroom" list item to the submenu and not modifying the open/close functionality of the submenu. Why do we need this method then?

Reply:
I added this 'classroom' option on the sidebar with 'Basic Mathematics' option in the submenu. So, that is why I used this toggle method to check if the submenu with option 'Basic Mathematics' is closed then on the another click it will be open.

Reply:
Ah, I see. Thanks for explaining.

Reply:
Done!